packages/block-editor-tools/README.md

```diff
@@ -10,8 +10,8 @@ This package contains a set of modules used by [Alley Interactive](https://alley
 - [Usage](#usage)
 - [Changelog](#changelog)
 - [Development Process](#development-process)
-    - [Contributing](#contributing)
-	- [Releases](#Releases)
+  - [Contributing](#contributing)
+- [Releases](#releases)
```

Comment:
Releases is at the same level as contributing, under development processes. I'd actually recommend just removing this entirely, as GitHub now has a table of contents for markdown files with headings bulit-in, and this takes up space and is redundant. I'll create a ticket to update our boilerplate to remove the TOC from our standard readmes.